
--- src/main/java/com/redhat/coolstore/rest/RestApplication.java
+++ src/main/java/com/redhat/coolstore/rest/RestApplication.java
@@ -1,5 +1,5 @@
 package com.redhat.coolstore.rest;

-import javax.ws.rs.ApplicationPath;
-import javax.ws.rs.core.Application;
+import jakarta.ws.rs.ApplicationPath;
+import jakarta.ws.rs.core.Application;

 @ApplicationPath("/services")
 public class RestApplication extends Application {
 }
